0011-revise-e2e-test-framework.adoc

```diff
@@ -314,6 +314,8 @@ This includes:
 3. Don't rely on the assertion message to debug what went wrong, instead check the logs for failing tests
 4. Split up logs into individual files per test: Use https://github.com/Jezorko/smack-my-jasmine-up to initialize the log framework with a different log file per test. PoC:
 
+NOTE: In a CI environment, you can then download these logs to your computer and inspect them.
```

Comment:
Yes but it's a pain.

Reply:
Would you rather look at a bazillion lines of log messages in stdout? :D

Reply:
Maybe... :D